src/develop/payment.md

```diff
@@ -0,0 +1,334 @@
+```
+  </TabItem>
+</Tabs>
+
+Next, create a file named `codegen.ts` in your project's root folder containing the following (adjust as needed):
+
+```typescript
+import type {CodegenConfig} from '@graphql-codegen/cli'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
+
+const schemaUrl = `https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: [
+    {
+      [schemaUrl]: {
+        headers: {
+          Authorization: `Basic ${PROJECT_SECRET_BASE64}`,
+        },
+      },
+    },
+  ],
+  documents: ['src/**/*.ts'],
+  ignoreNoDocuments: true, // for better experience with the watcher
+  generates: {
+    'src/generated/graphql.ts': {
+      plugins: ['typescript'],
+    },
+  },
+  watch: true,
+}
+
+export default config
+```
+
+Finally, follow [these instructions](https://the-guild.dev/graphql/codegen/docs/getting-started/development-workflow) to add GraphQL code generation to your development workflow.
+
+## API initialization
+
+To start making queries to the GraphQL API, you'll need to configure your client. Following is an example on how you can do that using either the Everscale SDK or Axios.
+
+<Tabs>
+  <TabItem value="sdk" label="Everscale SDK" default>
+
+```typescript
+import {TonClient} from '@eversdk/core'
+import {libNode} from '@eversdk/lib-node'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
+
+TonClient.useBinaryLibrary(libNode)
+
+const client = new TonClient({
+  network: {
+    endpoints: [`https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`],
+    access_key: PROJECT_SECRET_BASE64,
+  },
+})
+```
+  </TabItem>
+  <TabItem value="axios" label="Axios">
+
+```typescript
+import axios from 'axios'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
+
+axios.defaults.baseURL = `https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`
+axios.defaults.headers.post['Content-Type'] = 'application/json'
+if (PROJECT_SECRET_BASE64) {
+  axios.defaults.auth = {
+    username: '',
+    password: PROJECT_SECRET_BASE64,
+  }
+}
+```
+  </TabItem>
+</Tabs>
+
+## Code examples
+
+Following examples demonstrate how you can use the types generated from the GraphQL schema in your project.
+
+### Get account balance
+
+<Tabs>
+  <TabItem value="sdk" label="Everscale SDK" default>
+
+```typescript
+import {BlockchainQuery} from './generated/graphql'
+
+// Specify your account's address
+const ACCOUNT_ADDRESS = ''
+
+try {
+  const query = `
+    query {
+      blockchain {
+        account(
+          address: "${ACCOUNT_ADDRESS}"
+        ) {
+          info {
+            balance(format: DEC)
+          }
+        }
+      }
+    }`
+  const {result} = await client.net.query({query})
+  const blockchain: BlockchainQuery = result.data.blockchain
+  console.log(`The account balance is ${blockchain.account.info.balance / 10**9}`)
```

Comment:
Please handle `'blockchain.account.info' is possibly 'null' or 'undefined'`, for example:
```typescript
const balance = parseInt(blockchain.account?.info?.balance || '0', 10) / 10**9
console.log(`The account balance is ${balance}`)
```
(in other examples, too)